src/data/data_entry_flow.ts

```diff
@@ -28,6 +28,7 @@ export interface DataEntryFlowStepForm {
   data_schema: HaFormSchema[];
   errors: Record<string, string>;
   description_placeholders: Record<string, string>;
+  last_Step: boolean | null;
```

Comment:
```suggestion
  last_step: boolean | null;
```

Reply:
lol, I literally added 1 line and still messed up 🤦